agent/config/runtime_test.go

```diff
@@ -3588,6 +3590,7 @@ func TestFullConfig(t *testing.T) {
 		TelemetryAllowedPrefixes:                    []string{"oJotS8XJ", "consul.consul"},
 		TelemetryBlockedPrefixes:                    []string{"cazlEhGn"},
 		TelemetryMetricsPrefix:                      "ftO6DySn",
+		TelemetryPrometheusRetentionTime:            15 * time.Second,
```

Comment:
Why a default of 15 seconds when the lib defaults is 60 seconds ? (and it also looks like this expiration time should always be 0 or something, but I'm not sure how go-metrics work)

Reply:
This is only for testing that both HCL and JSON do take into account values, only for Unit Tests

Reply:
it is just a unit test to ensure the parameter is taken into account from hcl as well as json